wallettool/build.gradle

```diff
@@ -9,7 +9,10 @@ plugins {
 }
 
 def annotationProcessorMinVersion = GradleVersion.version("4.6")
-boolean hasAnnotationProcessor = (GradleVersion.current().compareTo(annotationProcessorMinVersion) > 0)
+boolean hasAnnotationProcessor = (GradleVersion.current().compareTo(annotationProcessorMinVersion) >= 0)
+
+def toolChainsMinVersion = GradleVersion.version("6.8")     // Toolchains with selection by vendor
+boolean hasToolchains = (GradleVersion.current().compareTo(toolChainsMinVersion) >= 0)
```

Comment:
Is "toolchains" a good name for this? What toolchain is it exactly? Besides, is it "toolchains" or "toolChains"?

Reply:
https://docs.gradle.org/current/userguide/toolchains.html

Toolchains were added in Gradle 6.7, but Gradle 6.8 added the JvmVendorSpec which is essential to using them in our (and most other) cases.

Reply:
So `toolChainsMinVersion` and `hasToolchains` may not be the best choice of name. The `hasToolchains` `boolean` currently "means" three things.


1. Has "Java Toolchain" support (Gradle 6.7+)
2. Has `JvmVendorSpec` support (Gradle 6.8+) in toolchains: see https://docs.gradle.org/6.8/release-notes.html#java-toolchain-improvements
3. Has "everything needed" for running the GraalVM task.

Reply:
Sory, I forgot about the "toolchain" feature. I guess the name is fine then, but lets stay with the non-camelcase spelling like in your linked doc.